app/views/results.nunjucks

```diff
@@ -25,7 +25,7 @@
 <div class="results__title">
   <h1>Pharmacies near {{ getDisplayLocation() }}</h1>
 </div>
-<div class="grid-row bankHoliday">
+<div class="grid-row {% if service.bankHolidayMessage %}bankHoliday{% endif %}">
```

Comment:
Should this class be added to the div for the entire result set?

Reply:
> Should this class be added to the div for the entire result set?

Ahhh good point! I put it in so that I could adjust the height of the `<li>`s if there is a bank holiday message. But Obviously, not all will have the message. I'll change it.